habitat-baselines/habitat_baselines/rl/ddppo/policy/resnet_policy.py

```diff
@@ -91,8 +92,22 @@ def from_config(
         action_space,
         **kwargs,
     ):
+        # Exclude cameras for rendering from the observation space.
+        ignore_names = [
+            config.TASK_CONFIG.SIMULATOR[k].UUID
```

Comment:
I doubt this will work

Reply:
Yes, unfortunately I am not done with the full training run through tests. I have not yet tested evaluation. So you're right, that code would not work.